### DISCUSSION TITLE
documentation of how emission factors for different industry classifications were derived

### DISCUSSION THREAD

**Discussion by hajarpb** [USEPA/supply-chain-factors#17] [Q&A] [answered]
Hi Mo, 
Is there any documentation of how emission factors for different industry classifications were derived?
I am specifically looking for "Scenic and sightseeing transportation and support activities for transportation". 

Thanks 
Hajar

**Comment by MoLi7** [comment, marked as answer]
Hi @hajarpb, detailed documentation of the 'how' can be found in [Appendix 1 of the Supply Chain GHG Emission Factors report](https://cfpub.epa.gov/si/si_public_file_download.cfm?p_download_id=540831&Lab=CESER). Essentially, the factors are derived through matrix algebra, which means all industries/commodities are presented in one matrix (e.g. _A_) and are passed the same method matrix multiplication. Therefore, the method for deriving factors for "Scenic and sightseeing transportation and support activities for transportation" is no different from the method used for other industries/commodities.

More documentation about what data are used for deriving the factors can be found in [model specifications in this repository](https://github.com/USEPA/supply-chain-factors/tree/main/model-specs). Taking USEEIOv2.0.16-GHG as an example, [economic data are from BEA](https://github.com/USEPA/supply-chain-factors/blob/main/model-specs/USEEIOv2.0.16-GHG.yml#L2-L9) and [emission data are from EPA's GHG Inventory](https://github.com/USEPA/supply-chain-factors/blob/main/model-specs/USEEIOv2.0.16-GHG.yml#L27-L33).

**Reply by WesIngwersen** [reply]
I would more broadly suggest that the 'how' is described in the Methods section of the [report](https://cfpub.epa.gov/si/si_public_record_Report.cfm?dirEntryId=349324). 

More simply stated though and to echo @MoLi7's response, the underlying methodology is the same for all commodities and industries.  The strength of this approach is in part providing this established methodology based on national statistics that is objective, provides a high level of comparability and therefore does not create a bias toward any particular commodity or industry.

**Comment by hajarpb** [comment]
Thank you very much for your time to provide me this detail. This is very
helpful.
Cheers
Hajar
